app/src/main/java/net/kuama/documentscanner/viewmodels/CropperModel.kt

```diff
@@ -21,6 +21,7 @@ class CropperModel : ViewModel() {
     val corners = MutableLiveData<Corners>()
     val originalBitmap = MutableLiveData<Bitmap>()
     val bitmapToCrop = MutableLiveData<Bitmap>()
+    val errors = MutableLiveData<String>()
```

Comment:
I'd prefer to have a `LiveData<Error>` instead of a string, so in the activity you can choose different behaviour based on the error, even if it is the same error